packages/dev/core/src/Shaders/ShadersInclude/pbrClusteredLightingFunctions.fx

```diff
@@ -0,0 +1,162 @@
+            , sheenOutParams sheenOut
+        #endif
+        #ifdef CLEARCOAT
+            , clearcoatOutParams clearcoatOut
+        #endif
+    ) {
+        float NdotV = absEps(dot(N, V));
+#include<pbrBlockReflectance0>
+        #ifdef CLEARCOAT
+            specularEnvironmentR0 = clearcoatOut.specularEnvironmentR0;
+        #endif
+
+        lightingInfo result;
+        ivec2 tilePosition = ivec2(gl_FragCoord.xy * lightData.xy);
+        int maskHeight = int(lightData.z);
+        tilePosition.y = min(tilePosition.y, maskHeight - 1);
+
+        int numBatches = (numLights + CLUSTLIGHT_BATCH - 1) / CLUSTLIGHT_BATCH;
+        int batchOffset = 0;
+
+        for (int i = 0; i < numBatches; i += 1) {
+            uint mask = uint(texelFetch(tileMaskTexture, tilePosition, 0).r);
+            tilePosition.y += maskHeight;
+
+            while (mask != 0u) {
+                // This gets the lowest set bit
+                uint bit = mask & -mask;
+                mask ^= bit;
+                int position = onlyBitPosition(bit);
+                SpotLight light = getClusteredSpotLight(lightDataTexture, batchOffset + position);
+
+                preLightingInfo preInfo = computePointAndSpotPreLightingInfo(light.vLightData, V, N, posW);
+                preInfo.NdotV = NdotV;
+
+                // Compute Attenuation infos
+                preInfo.attenuation = computeDistanceLightFalloff(preInfo.lightOffset, preInfo.lightDistanceSquared, light.vLightFalloff.x, light.vLightFalloff.y);
+                // Assume an angle greater than 180º is a point light
+                if (light.vLightDirection.w >= 0.0) {
+                    preInfo.attenuation *= computeDirectionalLightFalloff(light.vLightDirection.xyz, preInfo.L, light.vLightDirection.w, light.vLightData.w, light.vLightFalloff.z, light.vLightFalloff.w);
+                }
+
+                preInfo.roughness = adjustRoughnessFromLightProperties(reflectivityOut.roughness, light.vLightSpecular.a, preInfo.lightDistance);
+                preInfo.diffuseRoughness = reflectivityOut.diffuseRoughness;
+                preInfo.surfaceAlbedo = surfaceAlbedo;
+                lightingInfo info;
+
+                // Diffuse contribution
+                #ifdef SS_TRANSLUCENCY
+                    #ifdef SS_TRANSLUCENCY_LEGACY
+                        info.diffuse = computeDiffuseTransmittedLighting(preInfo, light.vLightDiffuse.rgb, subSurfaceOut.transmittance);
+                        info.diffuseTransmission = vec3(0);
+                    #else
+                        info.diffuse = computeDiffuseLighting(preInfo, light.vLightDiffuse.rgb) * (1.0 - subSurfaceOut.translucencyIntensity);
+                        info.diffuseTransmission = computeDiffuseTransmittedLighting(preInfo, light.vLightDiffuse.rgb, subSurfaceOut.transmittance);
+                    #endif
+                #else
+                    info.diffuse = computeDiffuseLighting(preInfo, light.vLightDiffuse.rgb);
```

Comment:
`info.diffuseTransmission = vec3(0);` is missing.

Reply:
`diffuseTransmission` is only defined and set to `vec3(0)` and used later  if `SS_TRANSLUCENCY` is defined. This `#else` branch here is where that isn't defined